web/src/components/Wrapper.js

```diff
@@ -1,5 +1,8 @@
+/* eslint-disable no-restricted-globals */
```

Comment:
What do you think about using `document.location` instead of just `location`?  Then we can get rid of this eslint exception.

Reply:
👯 I didn't realize `document.location` was accessible from here! 👯